sw/device/lib/testing/test_status.h

```diff
@@ -19,46 +19,58 @@
 typedef enum test_status {
   /**
    * Indicates that the CPU has started executing the boot_rom code.
+   *
+   * Writing this value to #kDeviceTestStatusAddress must not stop simulation of
```

Comment:
Is there a reason this is written #kConst and not `kConst` like the rest of our comments?

Reply:
Doxygen will automatically link names written like this.